FakebookNotifications/FakebookNotifications.DataAccess/NotificationsRepo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FakebookNotifications.DataAccess.Models;
+using FakebookNotifications.DataAccess.Models.Interfaces;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FakebookNotifications.DataAccess
+{
+    public class NotificationsRepo: INotificationsRepo
```

Comment:
should use dependency inversion etc., but these classes should not exist yet. you could make their interfaces, but not the classes yet.